salt/log/handlers/fluent_mod.py

```diff
@@ -455,7 +436,7 @@ def _make_packet(self, label, timestamp, data):
         packet = (tag, timestamp, data)
         if self.verbose:
             print(packet)
-        return msgpack.packb(packet)
+        return msgpack.packb(packet, _msgpack_module=msgpack)
```

Comment:
What is the driver allowing the use of a different msgpack module. I think we should keep things consistent by always using the module imported in `utils.msgpack`